README.md

```diff
@@ -17,10 +16,78 @@ docker login ghcr.io
 Then you can deploy the application by running:
 
 ```
-docker compose up
```

Comment:
This should be `docker compose`.
`docker-compose` (note the dash) is the older version.